test/reline/test_face.rb

```diff
@@ -171,11 +160,16 @@ def test_invalid_rgb?
       assert_equal false, @config.send(:rgb_expression?, "#FFFFF")
     end
 
-    def test_format_to_sgr
+    def test_format_to_sgr_preserves_order
       assert_equal(
         "#{RESET_SGR}\e[37;41;1;3m",
         @config.send(:format_to_sgr, foreground: :white, background: :red, style: [:bold, :italicized])
       )
+
+      assert_equal(
+        "#{RESET_SGR}\e[37;1;3;41m",
+        @config.send(:format_to_sgr, foreground: :white, style: [:bold, :italicized], background: :red)
```

Comment:
@hasumikin I'm not sure if the original intention was to test `format_to_sgr`'s ordering. Feel free to correct it in a later PR.

Reply:
@st0012 
Thanks, I originally wanted to test `#define` method, not `#format_to_sgr`. Seems Shibata-san made test-unit-rr available so I will revive it later.
Thank you anyway! It was great help!!